cms/envs/aws.py

```diff
@@ -87,6 +87,21 @@
     DEFAULT_PRIORITY_QUEUE: {}
 }
 
+# Setup alternate queues, to allow access to cross-process workers
+ALTERNATE_QUEUE_ENVS = os.environ.get('ALTERNATE_WORKER_QUEUES', '').split()
```

Comment:
`SERVICE_VARIANT` is defined in this manner, and this variable needs to match the other environment's `SERVICE_VARIANT`, so defining them in the same place seemed logical.